app/database/queries/breeze_buddy/chat_session.py

```
@@ -0,0 +1,182 @@
+        VALUES (
+            $1,
+            COALESCE(
+                (SELECT MAX(idx) + 1
+                 FROM {CHAT_MESSAGE_TABLE}
+                 WHERE session_id = $1),
+                0
+            ),
+            $2, $3
+        )
+        RETURNING {_MESSAGE_COLUMNS}
+    """
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)